api/src/test/scala/hmda/api/http/ValidationErrorConverterSpec.scala

```diff
@@ -17,17 +17,17 @@ class ValidationErrorConverterSpec extends WordSpec with MustMatchers with Valid
       val ctx = ValidationContext(None, Some(2017))
       val errors = badLars.flatMap(lar => validationErrors(lar, ctx, validateLar).errors)
       val syntacticalEditResults =
-        validationErrorsToEditResults(errors, Syntactical)
+        validationErrorsToEditResults(Nil, errors, Syntactical)
```

Comment:
With this update, this spec file needs an additional test to show that Transmittal Sheet errors convert correctly.